config.toml

```
@@ -24,7 +24,7 @@ theme = "qbs"
         weight = -105
 
     [[menu.main]]
-        identifier = "getqbs"
-        name = "Get Qbs"
-        url = "/getqbs"
+        identifier = "installing"
+        name = "Installing"
```

```suggestion
        identifier = "getqbs"
        name = "Get Qbs"
```

let's keep it this way, since imperative "get X" stuff is usually preferred for promo-y sites